client/src/modules/tags/tags.ctrl.js

```diff
@@ -0,0 +1,92 @@
+angular.module('bhima.controllers')
+  .controller('TagsController', TagsController);
+
+TagsController.$inject = [
+  '$uibModal', 'TagsService', 'SessionService', 'ModalService',
+  'NotifyService', 'bhConstants', 'uiGridConstants', '$rootScope',
+];
+
+function TagsController($uibModal, Tags, Session, Modal,
```

Comment:
It doesn't look like you use `Session`.  Please check all your imports and remove the ones you do not use.